modules/apps/bookmarks/bookmarks-service/service.xml

```diff
@@ -17,22 +17,22 @@
 
 		<column name="companyId" type="long" />
 		<column name="userId" type="long" />
-		<column name="userName" type="String" />
+		<column name="userName" type="String" uad-anonymize-field-name="fullName" />
 		<column name="createDate" type="Date" />
 		<column name="modifiedDate" type="Date" />
 
 		<!-- Other fields -->
 
 		<column name="folderId" type="long" />
 		<column name="treePath" type="String" />
-		<column name="name" type="String" />
-		<column name="url" type="String" />
-		<column name="description" type="String" />
+		<column name="name" type="String" uad-nonanonymizable="true" />
+		<column name="url" type="String" uad-nonanonymizable="true" />
+		<column name="description" type="String" uad-nonanonymizable="true" />
 		<column name="priority" type="int" />
 		<column name="lastPublishDate" type="Date" />
 		<column name="status" type="int" />
 		<column name="statusByUserId" type="long" />
-		<column name="statusByUserName" type="String" />
+		<column name="statusByUserName" type="String" uad-anonymize-field-name="fullName" />
```

Comment:
@julschong Line 20 and Line35 is adding same value `uad-anonymize-field-name="fullName"`, this does not look right to me

Reply:
@tinatian Hi Tina, I think you are right here. I see the the original non generated code is anonymizing using `anonymousUser.getScreenName()` I will check again and resend